web/app/scripts/custom-reports/custom-report-partial.html

```diff
@@ -7,14 +7,20 @@
                 <span>{{ ::ctl.params.occurred_max | localDateTime : ctl.dateFormat }}</span>
             </h2>
             <h3>
-                Incidents
-                <span ng-if="ctl.params.aggregation_boundary"> for {{ ::ctl.report.table_labels[table.tablekey] }}</span>
-                by {{ ::ctl.rowCategoryLabel }} and {{ ::ctl.colCategoryLabel }}
+                {{ ctl.recordType.plural_label }}
+                <span ng-if="ctl.params.aggregation_boundary">
+                  {{ 'COMMON.FOR' | translate }}
+                  {{ ::ctl.report.table_labels[table.tablekey] }}
+                </span>
+                {{ 'COMMON.BY' | translate }}
```

Comment:
This may need to be revisited; "by" is a weird word that has a lot of different meanings in different contexts. I'm not sure how Arabic handles things, but at least in the languages I know, "by" tends not to have a clean 1-to-1 mapping with another word (and it doesn't always map to a word at all, sometimes it's a prefix / suffix to another word).